src/index.js

```diff
@@ -3,12 +3,21 @@ import ReactDOM from "react-dom";
 import App from "./App";
 import { makeServer } from "./server";
 import { BrowserRouter } from "react-router-dom";
+import { FilterProvider } from "./context/filter-provider";
+import { ProductProvider } from "./context/product-provider";
+import { CategoryProvider } from "./context/category-provider";
```

Comment:
u can also make the index.js file in context folder to export the components or pages or providers